traitsui/testing/tester/_ui_tester_registry/qt4/_interaction_helpers.py

```diff
@@ -217,18 +217,19 @@ def mouse_click_combobox(combobox, index, delay):
         Time delay (in ms) in which each key click in the sequence will be
         performed.
     """
-    q_model_index = combobox.model().index(index, 0)
-    check_q_model_index_valid(q_model_index)
-    mouse_click_item_view(
-        model=combobox.model(),
-        view=combobox.view(),
-        index=q_model_index,
-        delay=delay,
-    )
-    # Otherwise the click won't get registered.
-    key_click(
-        combobox.view().viewport(), key="Enter", delay=delay
-    )
+    if combobox:
+        q_model_index = combobox.model().index(index, 0)
+        check_q_model_index_valid(q_model_index)
+        mouse_click_item_view(
+            model=combobox.model(),
+            view=combobox.view(),
+            index=q_model_index,
+            delay=delay,
+        )
+        # Otherwise the click won't get registered.
+        key_click(
+            combobox.view().viewport(), key="Enter", delay=delay
+        )
```

Comment:
IIRC, the plan here was to log a warning (or something on those lines) in an `else` block for the `if combobox` - to ensure that we're not quietly handling the case and potentially supressing coding mistakes.